_commands/tools/tmux.md

```diff
@@ -1,47 +1,44 @@
 ---
-
 ---
 
-
-
 tmux
-
----
+--
 
 'tmux' allows a user to use 1 terminal window to access multiple separate terminal sessions. It lets you keep running them in the background by detaching them and then reattach them to a different terminal.
 
 ~~~ bash
-
 $ tmux
 $ tmux new -s myname
 $ tmux a -t myname
 $ tmux ls
 $ tmux kill-session -t myname
-
 ~~~
 
+<!--more-->
+
 ### Useful Options / Examples
 
-#### CTRL-B + % or CTRL-B + "
 
-##### % represents a vertical split, while " represents a horizontal split. This will split the terminal window into multiple terminal sessions. Use CTRL-B + arrow keys to navigate.
+#### `CTRL-B + % or CTRL-B + "`
+
+% represents a vertical split, while " represents a horizontal split. This will split the terminal window into multiple terminal sessions. Use CTRL-B + arrow keys to navigate.
```

Comment:
Could we make `%` and `CTRL-B + arrow keys` inline code sections using back-ticks (like I've done in this review)?